packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-node-ciemss.vue

```diff
@@ -210,8 +210,11 @@ const preparedCharts = computed(() => {
 			}
 		);
 		applyForecastChartAnnotations(chart, annotations);
-		chart.layer.push(...createInterventionChartMarkers(groupedInterventionOutputs.value[variable]));
-
+		Object.entries(groupedInterventionOutputs.value).forEach((item) => {
+			item[1].forEach((intervention) => {
+				chart.layer.push(...createInterventionChartMarkers([intervention], false));
+			});
+		});
```

Comment:
This is going to end up drawing all the interventions (variables + parameters) for each chart. 
```suggestion
chart.layer.push(...createInterventionChartMarkers(groupedInterventionOutputs.value[variable]));
```
Don't we need to revert back to the previous code like ^?